src/main/java/org/agbrothers/englishbot/messagebuilder/AddWordMessageBuilder.java

```diff
@@ -0,0 +1,35 @@
+package org.agbrothers.englishbot.messagebuilder;
+
+import org.agbrothers.englishbot.entity.Word;
+import org.agbrothers.englishbot.service.DictionaryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static org.agbrothers.englishbot.constant.ButtonLabel.*;
+
+@Component
+public class AddWordMessageBuilder extends MessageBuilder{
+    private DictionaryService dictionaryService;
+
+    @Autowired
+    public void setDictionaryService(DictionaryService dictionaryService) {
+        this.dictionaryService = dictionaryService;
+    }
+
+    @Override
+    public String getResponseMessageText(String messageText) {
+        if (messageText.equals(ADD_WORD)){
+            return "Введіть слово англійською та його переклад через пробіл";
+        }
+        String[] wordData = messageText.split(" ");
+        if(wordData.length!=2 || !wordData[0].matches("^[a-zA-Z]+$") || !wordData[1].matches("^[А-ЩЬЮЯҐЄІЇа-щьюяґєії'`’ʼ]+$")){
+            return "Помилка в слові англійською або перекладі слова. Слово англійською має бути латиницею, переклад - лише кирилицею.";
+        }
+        if(null!=dictionaryService.getWordRepository().findByEnglishValue(wordData[0])) {
```

Comment:
NEVER do something like this!
You acces the repository via getter, instead you should add a method in service

It should be `dictionaryService.getWordByEnglishValue(wordData[0])`
 - remove getter for wordRepository in DictionaryService
 - add method `getWordByEnglishValue(wordData[0])` in DictionaryService which should return `wordRepository.findByEnglishValue(wordData[0])`